cmd/sops/main.go

```diff
@@ -207,32 +207,45 @@ func main() {
 				return cli.NewExitError("Error: cannot operate on non-existent file", exitNoFileSpecified)
 			}
 		}
-		fileBytes, err := ioutil.ReadFile(fileName)
-		if err != nil {
-			return err
-		}
 
 		inputStore := inputStore(c, fileName)
 		outputStore := outputStore(c, fileName)
 
-		tree, err := loadPlainFile(c, inputStore, fileName, fileBytes)
+		fileBytes, readFileErr := ioutil.ReadFile(fileName)
+		plainTree, loadPlainFileErr := loadPlainFile(c, inputStore, fileName, fileBytes)
+		encryptedTree, loadEncryptedFileErr := loadEncryptedFile(c, inputStore, fileBytes)
```

Comment:
This is weird. I think I understand why you did it this way, but it's a very unusual flow. At the very least, add a comment to explain why it's done this way.